stacks/BasicApiStack.ts

```diff
@@ -2,7 +2,18 @@ import { StackContext, Api, Table, Queue, Bucket, Topic, Config } from '@serverl
 import { SSTConstruct } from '@serverless-stack/resources/dist/Construct'
 
 export function BasicApiStack ({ app, stack }: StackContext): { queue: Queue, bucket: Bucket } {
-  const queue = new Queue(stack, 'Pin')
+  const dlq = new Queue(stack, 'Pin-Dlq')
```

Comment:
```suggestion
  const dlq = new Queue(stack, 'PinDlq')
```
Suggest to remove the hyphen (`-`) to be consistent with other resource names